src/index.html

```diff
@@ -5,8 +5,291 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#home">
+          <img src="images/logo.png" alt="logo">
```

Comment:
Remember about using more descriptive alt attributes (for SEO). "Moyo logo" would be much better ;)

Reply:
@marekder one thing I forgot about - add smooth transitions and animations. Check the readme

Reply:
Done.